tycho-client/src/deltas.rs

```diff
@@ -579,6 +595,7 @@ impl WsDeltasClient {
         subscription_id: Uuid,
         ready_tx: oneshot::Sender<()>,
     ) -> Result<(), DeltasError> {
+        debug!(?subscription_id, "Unsubscribing");
```

Comment:
Can we make this info?
Just to ensure this always appears in the log, specially when it's sent to us